api/http/pprof/pprof.go

```diff
@@ -12,31 +11,6 @@ import (
 func Enable(address string) {
 	http.Handle("/", http.RedirectHandler("/debug/pprof", http.StatusMovedPermanently))
 
-	http.Handle("/debug/vars", http.HandlerFunc(expVars))
-	http.Handle("/debug/pprof/", http.HandlerFunc(pprof.Index))
-	http.Handle("/debug/pprof/cmdline", http.HandlerFunc(pprof.Cmdline))
-	http.Handle("/debug/pprof/profile", http.HandlerFunc(pprof.Profile))
-	http.Handle("/debug/pprof/symbol", http.HandlerFunc(pprof.Symbol))
-	http.Handle("/debug/pprof/block", pprof.Handler("block"))
```

Comment:
`debug/pprof/block`, `debug/pprof/heap`, `debug/pprof/goroutine`, `debug/pprof/threadcreate` do not seem to be added by the init of `net/http/pprof`. Should probably left there